app/controllers/Application.scala

```diff
@@ -97,26 +102,26 @@ class Application(
     ))
   }
 
-  def newContributionsLanding(countryCode: String): Action[AnyContent] = CachedAction() { implicit request =>
-    Ok(views.html.main(
-      title = "Support the Guardian | Make a Contribution",
-      description = Some(stringsConfig.contributionsLandingDescription),
-      mainId = s"new-contributions-landing-page-$countryCode",
-      mainJsBundle = "newContributionsLandingPage.js",
-      mainStyleBundle = "newContributionsLandingPageStyles.css",
-      scripts = views.html.addToWindow("paymentApiPayPalEndpoint", paymentAPIService.payPalCreatePaymentEndpoint)
-    ))
+  def newContributionsLanding(countryCode: String): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
+    type Attempt[A] = EitherT[Future, String, A]
+    request.user.traverse[Attempt, IdUser](identityService.getUser(_)).fold(
+      _ => Ok(newContributions(countryCode, None)),
+      user => Ok(newContributions(countryCode, user))
+    )
   }
 
-  def newContributionsThankyou(countryCode: String): Action[AnyContent] = NoCacheAction() { implicit request =>
-    Ok(views.html.main(
-      title = "Support the Guardian | Thank you for your contribution",
-      description = None,
-      mainId = s"new-contributions-thank-you-page-$countryCode",
-      mainJsBundle = "newThankYouPage.js",
-      mainStyleBundle = "newContributionsLandingPageStyles.css"
-    ))
-  }
+  private def newContributions(countryCode: String, idUser: Option[IdUser])(implicit request: RequestHeader) =
+    views.html.oneOffContributions(
```

Comment:
I think it would be great to create a new dedicated "skeleton" page right now, since it's confusing seeing `oneOffContributions` being used as the shell for a page which can do both one-off and recurring

Reply:
Yes I'm doing that.